### DISCUSSION TITLE
Wi-fi Devices cannot be found

### DISCUSSION THREAD

**Discussion by MONKE121** [BornToBeRoot/NETworkManager#2580] [Q&A]
Hi, I found your application and installed it, thinking it could troubleshoot my Wi-Fi and LAN issues. I don't have knowledge in any codes or network issues. I found your application on GitHub, hoping somebody could help me with my problem.

My issue is that I want my PC and my phone to detect each other's IP address, which is not happening. My PC doesn't have a Wi-Fi adapter. Although I tried the IP Scanner in your application, it only shows two IP addresses: my PC and the Internet Service Provider router. On the other hand, I installed an application on my phone, Net Analyzer, which can detect all connected Wi-Fi IP addresses. However, it can only detect the Internet Service Provider router.

Is there a way to solve this problem? I just want to connect my phone for karaoke.

Thank you

**Comment by BornToBeRoot** [comment]
Hi, sorry but I can't give support for home networks here. 
I think it's best to ask your question here: https://www.reddit.com/r/HomeNetworking/

Either the devices are not in the same network (home, guest) or a (client) firewall is blocking this.
